### PR TITLE
feat: Content Block publish component generator

### PR DESCRIPTION
**Pull request by istarkov**
## Description

1 commit = fixtures
2nd commit = code changes
3rd = size changes

ref #3994

Publish result:
https://webstudio-fixture-project-a-0su3o.wstd.work/content-block

See fixtures for generated code.

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 0000)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: packages/react-sdk/src/component-generator.ts [hunk 1]
```diff
@@ -231,6 +236,10 @@ export const generateJsxElement = ({
   }
 
   let generatedElement = "";
+  if (instance.component === blockTemplateComponent) {
```
Comment (TrySound) [inline packages/react-sdk/src/component-generator.ts:239]: can you bail out on top of function near descendant function?

File: packages/react-sdk/src/component-generator.ts [hunk 2]
```diff
@@ -246,6 +255,10 @@ export const generateJsxElement = ({
     generatedElement += children;
     generatedElement += `</Fragment>\n`;
     generatedElement += `)}\n`;
+  } else if (instance.component === blockComponent) {
+    if (instance.children.length > 1) {
+      generatedElement += children;
+    }
```
Comment (TrySound) [inline packages/react-sdk/src/component-generator.ts:261]: no need to check instance.children. block template is managed already in the other place

**Comment by TrySound** [comment]
A would add a couple basic tests to easier find the expected result

File: packages/react-sdk/src/component-generator.ts (hunk 1 above)
Comment (istarkov) [reply packages/react-sdk/src/component-generator.ts:239]: why? here is all instance logic in a single place

File: packages/react-sdk/src/component-generator.ts (hunk 2 above)
Comment (istarkov) [reply packages/react-sdk/src/component-generator.ts]: Wow really, thanks

**Comment by istarkov** [comment]
Tests added

**Review by TrySound — APPROVED** [review]

**Pull request merged**